pkg/services/searchV2/bluge.go

```diff
@@ -254,8 +242,30 @@ var ngramQueryAnalyzer = &analysis.Analyzer{
 	},
 }
 
-func getNameNGramField(name string) bluge.Field {
-	return bluge.NewTextField(documentFieldName_ngram, name).WithAnalyzer(ngramIndexAnalyzer)
+// Names need to be indexed a few ways to support key features
+func newSearchDocument(uid string, name string, descr string, url string) *bluge.Document {
+	doc := bluge.NewDocument(uid)
+
+	if name != "" {
+		doc.AddField(bluge.NewTextField(documentFieldName, name).StoreValue().SearchTermPositions())
+		doc.AddField(bluge.NewTextField(documentFieldName_ngram, name).WithAnalyzer(ngramIndexAnalyzer))
+
+		// Don't add a field for empty names
+		sortStr := strings.Trim(strings.ToUpper(name), " ")
+		if len(sortStr) > 0 {
+			if sortStr[0] < 'A' {
+				sortStr = "_" + sortStr // put numbers and punctuation *after* A-Z
```

Comment:
hmm, it probably doesn't matter much, but don't numbers come before letters in almost every file explorer? I just checked macos finder and google drive and they both behave in the same way.

Reply:
true true -- I'll remove this, my index was full of a ton of things that start with `$` so it was pages and pages of data before I got to an "A" 😢

Reply:
haha I see... when `1111test` isn't enough :D would be also great to latinize the characters to support äóąę etc in both sorting and searching but that's for laaater